From: Alexander Larsson <alexl@redhat.com>
Date: Thu, 16 Jun 2016 01:31:17 +0200
Subject: sandbox: Make /var/tmp and /tmp different dirs, not symlinks

This means you can use either or both of --filesystem=/tmp and
--filesystem=/var/tmp to share the host tmpdirs.
---
 common/flatpak-run.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/flatpak-run.c b/common/flatpak-run.c
index 317c317..0161280 100644
--- a/common/flatpak-run.c
+++ b/common/flatpak-run.c
@@ -2828,10 +2828,10 @@ flatpak_run_setup_base_argv (GPtrArray      *argv_array,
             "--dev", "/dev",
             "--proc", "/proc",
             "--dir", "/tmp",
+            "--dir", "/var/tmp",
             "--dir", "/run/host",
             "--dir", run_dir,
             "--setenv", "XDG_RUNTIME_DIR", run_dir,
-            "--symlink", "/tmp", "/var/tmp",
             "--symlink", "/run", "/var/run",
             "--ro-bind", "/sys/block", "/sys/block",
             "--ro-bind", "/sys/bus", "/sys/bus",
